api/v3/Activity.php

```diff
@@ -341,6 +341,10 @@ function civicrm_api3_activity_get($params) {
   }
 
   $activities = _civicrm_api3_basic_get(_civicrm_api3_get_BAO(__FUNCTION__), $params, FALSE, 'Activity', $sql);
+  if ($options['is_count']) {
```

Comment:
Note that this change takes us from a fatal error when we do getcount + check_permissions to a count that does not properly apply permissions. I think this is arguably better & the count doesn't seem to be highly sensitive info.

More importantly I think it's going to take a few steps to resolve this & I need to be able to get changes that don't break the tests.

The next steps to resolve this issue are

1) resolve https://lab.civicrm.org/dev/core/issues/454 and
2) get this extraction reviewed & merged https://github.com/civicrm/civicrm-core/pull/12949 (I have a follow on for that commit still in the tidying realm)

Reply:
update the extraction in 2 is merged - now this is the current PR https://github.com/civicrm/civicrm-core/pull/12995

Reply:
Can we call Activity.getcount directly here?

Reply:
this is for when getcount has been called - we have to return here as the next loop hard fails if we don't :-( I'm working to improve the next loop but need this to get tests running  - it's moved up from 3 rows further down